src/EnvVarProcessor.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\DatadirTests;
+
+use Keboola\DatadirTests\Exception\EnvVariableNotFoundException;
+use Keboola\DatadirTests\Exception\UnexpectedTypeException;
+
+class EnvVarProcessor
+{
+    /**
+     * @return mixed
+     */
+    public function replaceEnv(string $value)
+    {
+        $pattern =
+            '~^' .                       // start
+            '%env\(' .                   // %env(
+            '(?P<type>[a-z0-9_\-]+):' .  // TYPE:
+            '(?P<var>[a-z0-9_\-]+)' .    // VAR
+            '\)%' .                      // )%
+            '$~i';                       // end
+
+        if (preg_match($pattern, $value, $m)) {
+            $var = $m['var'];
+            $value = $this->getEnv($var);
+            $value = $this->convertTo($m['type'], $var, $value);
+        }
+
+        return $value;
+    }
+
+    protected function getEnv(string $var): string
+    {
+        $value = getenv($var);
+        if ($value === false) {
+            throw new EnvVariableNotFoundException(sprintf('Env variable "%s" not found.', $var));
+        }
+
+        return $value;
+    }
+
+    /**
+     * @return mixed
+     */
+    protected function convertTo(string $type, string $var, string $value)
+    {
+        switch ($type) {
+            case 'string':
+                return $value;
+            case 'int':
+                return intval($value);
+            case 'float':
+                return floatval($value);
+            case 'bool':
+                return self::stringToBool($value);
+        }
+
+        throw new UnexpectedTypeException(sprintf('Unexpected type "%s" of the env variable "%s".', $type, $var));
+    }
+
+    protected static function stringToBool(string $value): bool
+    {
+        $value = strtolower($value);
```

Comment:
co když chci bool jako "nastaveno/nenastaveno"? 
```
"shouldUseDefaultHost" : "%env(bool:HOST)%"
```

Reply:
Ale jestli se stejně chová ta symfony implementace tak OK

Reply:
Osobně bych to otočil a vrátil false pro 

```
''
'f'
'false'
'no'
'0'
```
A true pro všechno ostatní

Reply:
ok, mozem to otocit, aby bol lubovolny string true, tak ako v PHP bezne.

Reply:
https://github.com/keboola/datadir-tests/pull/29/commits/2f5bdae1c5f209853af0fbee756821366c5b1e6c